CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
```

Comment:
Suggest removing this document, since we now have all the governance documents including `contributing.md` in `.github` repo.